scripts/west_commands/create_board/ncs_create_board.py

```diff
@@ -200,12 +213,31 @@ def do_run(self, args, unknown_args):
             with open(out_dir / f"{ name }_defconfig", "w") as f:
                 f.write(tmpl.render(target=target))
 
-            tmpl = env.get_template("board.dts.jinja2")
+            if not target["xip"]:
+                tmpl = env.get_template("board.dts.jinja2")
+            else:
+                tmpl = env.get_template("board-xip.dts.jinja2")
+
             with open(out_dir / f"{name}.dts", "w") as f:
                 f.write(tmpl.render(target=target))
 
             tmpl = env.get_template("board_twister.yml.jinja2")
             with open(out_dir / f"{name}.yml", "w") as f:
                 f.write(tmpl.render(target=target))
 
-        print(f"Board {input['board']} created successfully")
+        # return post-commands
+        commands = []
+
+        if series == "nrf54h":
+            commands.append(
+                {
+                    "name": "Create BICR",
+                    "command": "west",
+                    "args": ["ncs-bicr", "--board-dir", str(out_dir.resolve())],
```

Comment:
Could you append `-s` to the list of arguments to generate the schema by running the command directly, please?